ui/pages/Project/components/ProjectOverview.jsx

```diff
@@ -132,7 +132,7 @@ const ProjectOverview = React.memo((
   }
 
   let editIndividualsButton = null
-  if (user.isStaff) {
+  if (user.isPm || (project.hasCaseReview && project.canEdit)) {
```

Comment:
Maybe the condition should include `user.isDataManager`.
By the way, we have `manager` for a project and `PM` for all projects. The `PM` is also `project manager` which could cause confusing. How about change the meaning of `PM` to `Program manager` who manages all the projects in the seqr "program"?
Another question I have is why `PM` and `Data Manager` can't be merged into a single permission level?

Reply:
no, these are totally different people with totally different jobs, totally, different access, and, legally, different definitions of "privilege". Data managers will be Mike Wilson, Katherine Chao, and William Phu, who are responsible for seqr elasticsearch data loading. They will, legally speaking, be "privileged" users due to what they can do in the system. They very intentionally are NOT given access here, as this isn't something they should ever be changing. PMs are Stacey Hall and Kayla Delano, who are the Project Managers for seqr and manage family/ individual data in seqr. They do this work for logistical reasons related to the CMG. For non-CMG projects we may allow project managers to do this role. As such, this is not considered a "privileged" user role. We are absolutely not changing what PM stands for because it is literally those user's job title and the name of the Broad team they are a part of

Reply:
Just for confirmation, PMs and Data managers can access all projects (including non-CMG ones) for family/individual data and data loading, is that correct? It doesn't like the analysts who may not have access to some projects depending on the project category.

Reply:
Data mangers can access all projects, PMs can not. PMs will probably all also be analysts, but don't have to be.